http/README.md

```diff
@@ -35,10 +35,36 @@ actions {
         port = 3000
         allowedRequestHeaders = ["Content-Type"]
       }
+      responseOptions {
+        predicates = [JSON]
+        forceJson = false
+      }       
     }
   }
 }
 ```
 
+- `endpointOptions` config describes quite basic request parameters as `path`, `domain`, `port` and `allowedRequestHeaders` which
+are necessary to perform http request 
+- `responseOptions` config is responsible for handling incoming request properly. Here we can specify `predicates` - it's array
+containing Vert.x response predicates, to get more familiar with it, please visit [this page](https://vertx.io/blog/http-response-validation-with-the-vert-x-web-client/).
```

Comment:
```suggestion
containing Vert.x response predicates, to get more familiar with it, please visit [this page](https://vertx.io/blog/http-response-validation-with-the-vert-x-web-client/).
You may find all available predicates [here].(https://vertx.io/docs/apidocs/io/vertx/ext/web/client/predicate/ResponsePredicate.html)
```

Maybe we should emphasise somehow the values user can specify as `predicates`? I'm not sure if my suggestion above is enough.